.gitignore

```diff
@@ -0,0 +1,7 @@
+/gradlew
```

Comment:
Thanks for adding this.  I'll  remove the idea and gradle files/folders in  a separate PR.